selfdrive/test/process_replay/test_processes.py

```diff
@@ -6,17 +6,20 @@
 import tempfile
 
 from selfdrive.car.car_helpers import interface_names
-from selfdrive.test.process_replay.compare_logs import compare_logs
 from selfdrive.test.process_replay.process_replay import replay_process, CONFIGS
+from selfdrive.test.process_replay.compare_logs import compare_logs
 from tools.lib.logreader import LogReader
 
+
+INJECT_MODEL = 0
+
 segments = [
   ("HONDA", "0375fdf7b1ce594d|2019-06-13--08-32-25--3"),      # HONDA.ACCORD
   ("HONDA", "99c94dc769b5d96e|2019-08-03--14-19-59--2"),      # HONDA.CIVIC
-  ("TOYOTA", "cce908f7eb8db67d|2019-08-02--15-09-51--3"),     # TOYOTA.COROLLA_TSS2
-  ("GM", "7ad88f53d406b787|2019-07-09--10-18-56--8"),         # GM.VOLT
-  ("HYUNDAI", "704b2230eb5190d6|2019-07-06--19-29-10--0"),    # HYUNDAI.KIA_SORENTO
-  ("CHRYSLER", "b6e1317e1bfbefa6|2019-07-06--04-05-26--5"),   # CHRYSLER.JEEP_CHEROKEE
+  #("TOYOTA", "cce908f7eb8db67d|2019-08-02--15-09-51--3"),     # TOYOTA.COROLLA_TSS2
```

Comment:
Thought you needed to remove routes from test_car_models, which I'm fine with temporarily. However, we can't remove any from process_replay